Makefile

```diff
@@ -19,7 +19,7 @@ build-linux: pkger
 build-docker:
 	docker build -t publicawesome/stakewatcher .
 
-migrate: pkger
+run-migrate: pkger
```

Comment:
Any chance this change might have an effect on one our scripts?

Reply:
Good point, let me check...

Reply:
It doesn't affect, is just to locally test with automigrations

Reply:
Looks like we are using `--auto-migrate` in scripts instead of this make target